webapp/app/components/pages/navbar/LoginNavigation.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
I could not find the test for this component, does it have one?

Reply:
Done!